docs/platform/what-is-tizen/profiles/iot.md

```diff
@@ -0,0 +1,40 @@
+# Tizen IoT
+
+Tizen IoT profile allows developers to make various kinds of IoT devices and develop applications for the devices.
+Tizen IoT Headed and Headless binaries support any type of IoT devices based on the Linux kernel.
+
+## Development Environment for Tizen IoT
```

Comment:
Global: Please use sentence case for all the headings.

Line 6: ## Development environment for Tizen IoT
Line 18: ## Create Tizen IoT images

Reply:
I also made it consistent with the other pages. Could you confirm once more?